README.md

```diff
@@ -1,10 +1,16 @@
 # Entity version
 
-The Entity Version module attaches a version number to content entity revisions that helps content editors to understand the evolution of the content item. 
+The Entity Version module attaches a version number to content entity revisions that helps content editors to understand the evolution of the content item.
 Versions are composed of major, minor and patch numbers and can be changed manually or by third party modules.
 
 The module includes the "Entity version workflows" sub-module which allows configuration and control of version numbers through workflow state transitions.
 
+## Configurations
+It provides a configuration page found at "admin/config/entity-version/settings" where an appropriate Version field can be selected for entities and their bundles that have at least one Version field.
```

Comment:
Add that then this field will be used as target for functionalities provided by sub-modules.